envi/archs/arm/disasm.py

```diff
@@ -2587,6 +2591,146 @@ def getOperValue(self, op, emu=None):
     def repr(self, op):
         return "c%d"%self.val
 
+#copied code from ArmImmOffsetOper - works but render is not correct. Not sure if can delete commented code yet
+class ArmCoprocOption(ArmImmOffsetOper):
+    def __init__(self, base_reg, offset, va, pubwl=8):
+        self.base_reg = base_reg
+        self.offset = offset
+        self.pubwl = pubwl
+        self.va = va
+        b = (pubwl >> 2) & 1
+        self.tsize = (4,1)[b]
+    '''
+    def setOperValue(self, op, emu=None, val=None):
```

Comment:
definitely need to override get/setOper*.  perhaps it's just a check for "option" mode, then return a call to ArmImmOffsetOper.get/set/Oper*()